### PR TITLE
New Websites SDK test recordings, assembly version update

### PR DESCRIPTION
**Pull request by nking-1**
Updated the Websites SDK test recordings so that the tests will pass. Also updated the assembly version to 2.1.0 -> 2.2.0.

### CODE REVIEW

File: src/SDKs/WebSites/Management.Websites/Microsoft.Azure.Management.Websites.csproj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Websites</PackageId>
 		<Description>Provides website management capabilities for Microsoft Azure.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Websites</AssemblyName>
-		<Version>2.1.0</Version>
+		<Version>2.2.0</Version>
 		<PackageTags>Microsoft Azure website management;website management;</PackageTags>
 		<PackageReleaseNotes><![CDATA[This is the Azure App Service SDK for .NET, based on the Azure App Service REST API, version 2018-11-01 of Certificates, and 2018-02-01 of all other resources]]></PackageReleaseNotes>
```
Comment (dsgouda) [inline src/SDKs/WebSites/Management.Websites/Microsoft.Azure.Management.Websites.csproj]: Please update `PackageReleaseNotes` here

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/WebSites/Management.Websites/Microsoft.Azure.Management.Websites.csproj (hunk 1 above)
Comment (Nking92) [reply src/SDKs/WebSites/Management.Websites/Microsoft.Azure.Management.Websites.csproj]: Updated the notes.

**Comment by Nking92** [comment]
@dsgouda Please merge if this looks good now.

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**